tests/org.jboss.tools.docker.ui.bot.test/src/org/jboss/tools/docker/ui/bot/test/image/PushImageTest.java

```diff
@@ -49,6 +50,13 @@ public void before() {
 
 	@Test
 	public void pushImage() {
+		String dockerHubUsername = System.getProperty("dockerHubUsername");
+		String dockerHubEmail = System.getProperty("dockerHubEmail");
+		String dockerHubPassword = System.getProperty("dockerHubPassword");
+		if (dockerHubUsername == null || dockerHubUsername.isEmpty() || dockerHubEmail == null
+				|| dockerHubEmail.isEmpty() || dockerHubPassword == null || dockerHubPassword.isEmpty()) {
+			fail("At least one of credentials is null or empty! Aborting test...");
```

Comment:
I think it would be good to list those properties in fail message. If somebody tries to run this test and it fails here he wouldn't know what possible arguments are required. No need to check every parameter by itself and be specific what parameter is missing, but it wouldn't hurt ;-)